static/js/chart_config.json

```diff
@@ -192,25 +192,25 @@
             "title": "Median Income",
             "topic": "Income",
             "chartType": "LINE",
-            "statsVars": ["MedianIncome"],
+            "statsVars": ["Median_Income_Person"],
             "unit": "$",
             "source": "US Census",
             "url": "https://www.census.gov/"
           },
           {
-            "title": "Median Income Trend",
```

Comment:
@shifucun @beets wanted to bring your attention to this-- I corrected this as I think it is a typo but want to check with y'all.

Reply:
Looks good, thx!